test/unit/features/directives/pre.spec.js

```diff
@@ -42,4 +42,15 @@ describe('Directive v-pre', function () {
     vm.$mount()
     expect(vm.$el.firstChild.tagName).toBe('VTEST')
   })
+
+  // #10087
+  it('should not compile attributes', function () {
+    Vue.component('vtest', { template: ` <div>Hello World</div>` })
```

Comment:
I don't think the test requires a component usage here, we could use a raw p or div